libkbfs/quota_usage.go

```diff
@@ -0,0 +1,134 @@
+// Copyright 2017 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package libkbfs
+
+import (
+	"sync"
+	"sync/atomic"
+	"time"
+
+	"golang.org/x/net/context"
+
```

Comment:
That was `goimports` when I changed `context` to `golang.org/x/net/context` I think. I removed space here and it happily rearranged it to have only two blocks. Will update!